### PR TITLE
Fix WalletConnect header alignment on Android

### LINKED ISSUE #20759: WalletConnect request header UI is misaligned
The `Account` element is not aligned in the header on the signing page

Screenshot 2024-07-15 at 15 44 21

**Steps:**
1. Open a DApp, e.g. https://opensea.io/ (reproducible for all dapps)
2. Scan QR code to connect your wallet
3. Press `Connect` on the confirmation page
4. Check the icon on the next screen 

_Originally posted by @qoqobolo in https://github.com/status-im/status-mobile/issues/20693#issuecomment-2228549019_

### LINKED ISSUE #20859: 🎴 Long dApp names are broken within request modal
Reported in #20811 

Here is an example from OpenSea https://opensea.io/

photo_2024-07-22 19 38 44

### PR DESCRIPTION
**Pull request by clauxx**
fixes #20759
fixes #20859

### Summary

Fixed alignment issues with the wc header, which are mostly noticeable on Android. Also, added max width (via container-style) to quo/summary-tag, so it never goes out of the screen.

### Testing notes
Make sure summary/tag usage is not broken, cause I added some small changes there. 

### Steps to test

- Open Status on Android
- Send any WC request
- Observe the header

### Screenshots

Screenshot 2024-07-23 at 15 24 00
Screenshot 2024-07-23 at 15 23 31

status: ready

### CODE REVIEW

File: src/quo/components/tags/summary_tag/style.cljs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 (defn main
   [{:keys [type customization-color]} theme]
-  {:justify-content  :center
+  {:justify-content  :flex-start
```
Comment (shivekkhurana) [inline src/quo/components/tags/summary_tag/style.cljs:7]: Have you confirmed that other usage of tags are not affected ?

Comment (clauxx) [reply src/quo/components/tags/summary_tag/style.cljs:7]: I checked the preview, but that's about it

Comment (shivekkhurana) [reply src/quo/components/tags/summary_tag/style.cljs:7]: It might be an issue, let's add a testing note so QA can ensure nothing breaks ?

**Review by shivekkhurana — APPROVED** [review]

**Comment by shivekkhurana** [comment]
Also fixes https://github.com/status-im/status-mobile/issues/20859

File: src/quo/components/tags/summary_tag/style.cljs (hunk 1 above)
Comment (clauxx) [reply src/quo/components/tags/summary_tag/style.cljs:7]: it's there already

**Review by alwx — APPROVED** [review]

**Comment by clauxx** [comment]
@qoqobolo @pavloburykh ready for testing

**Comment by mariia-skrypnyk** [comment]
Hey @clauxx !

Thanks for PR.

- Header alignment is fixed and looks good 👍

- Here are some leftovers after Long name fix and others:

**1.** The part of title is missing: "...the largest NFT market"
**Android** and **iOS**
Screenshot 2024-07-25 at 09 50 37 Screenshot 2024-07-25 at 09 53 22
___________________________________________
**2.** Specific cut of letters that has long lover part (g,j etc)
Screenshot 2024-07-25 at 09 55 43
**Device:** Samsung A34 (14.0)
_____________________________________________
**3.** iOS long name shortening looks too close to the right boarder
Screenshot 2024-07-25 at 09 57 16
**Device:** iPhone 13 (17.5.1)

**Comment by clauxx** [comment]
@mariia-skrypnyk 

_1_ and _3_ are fixed and can be checked again.

_2_ seems to not be specific to the android device here, as on the Pixel it's fine. I think the cut off "g" issue can be logged separately with a low priority. 

Also I notice the same thing with the "g" being cut off on the Connection screen (screenshot issue 1 on android), so I think this may be an issue with the `standard-title` component on this device. Did you notice it in other places e.g. settings headers (privacy&security/network) or the preview for `standart-title`?

**Comment by mariia-skrypnyk** [comment]
Hi @clauxx !

Despite the fact that we have a strange issue with modals now I had a chance to check your fixes.

1 👍
3 👍

2 - yes, this cut also is present in the whole app and I will log the separate issue.

PR can be merged!

**Pull request merged**